src/service/pending-storage-service.js

```diff
@@ -6,18 +6,27 @@ class PendingStorageService {
         this.ualService = ctx.ualService;
     }
 
-    async cacheAssertion(repository, blockchain, contract, tokenId, assertion, operationId) {
+    async cacheAssertion(
+        repository,
+        blockchain,
+        contract,
+        tokenId,
+        stateId,
```

Comment:
Please use consistent naming. There currently is state, stateId and assertionId throughout the code and they are all used to represent the same thing.

Reply:
Renamed to asssertionId everywhere for now